cmd/virt-prerunner/main.go

```diff
@@ -503,10 +510,15 @@ func startDHCPServer(ifaceName string, mac net.HardwareAddr, ipNet *net.IPNet, g
 		"mac":          mac.String(),
 		"ip":           ipNet.IP.String(),
 		"mask":         net.IP(ipNet.Mask).String(),
-		"gateway":      gateway.String(),
+		"routes":       formatAndSortRoutes(routes),
```

Comment:
If routes is an empty array, the result would be an empty string? Is it expected?

Reply:
a pod interface with IPAM enable, it will at least have one route. If IPAM disabled will not start a dhcp server. to make it robust, I just test with  empty routes, the dhcp server still works with `dhcp-option=option:classless-static-route,`